### PR TITLE
feat(docs): Add RabbitMQ Metrics and Logs doc

### PR DESCRIPTION
**Pull request by crazyuploader**
## Pull Request

---

### 📄 Summary
This PR adds doc for RabbitMQ logs and metrics

#### Issues closed by this PR
Closes https://github.com/SigNoz/growth-pod/issues/1057
Closes https://github.com/SigNoz/growth-pod/issues/818

Doc: [Link](https://staging.signoz.io/docs/integrations/opentelemetry-rabbitmq)

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

---

- Tests added/updated:
- Manual verification:
- Edge cases covered:

---

- Blast radius:
- Potential regressions:
- Rollback plan:

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Feature / Bug Fix / Maintenance |
| Description | User-facing summary |

---

### 📋 Checklist
- [ ] Tests added or explicitly not required
- [ ] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

---

## 👀 Notes for Reviewers

---

### CODE REVIEW

File: data/docs/integrations/opentelemetry-rabbitmq.mdx [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+
+<Admonition type="note">
+  Keep port **15672** off the public network. Use HTTPS with certificate verification if the Collector connects across hosts. See the RabbitMQ <a href="https://www.rabbitmq.com/docs/management#monitoring-only-user" target="_blank" rel="noopener noreferrer nofollow">monitoring-only user</a> guidance.
+</Admonition>
+
+### Step 2: Configure and run the Collector
+
+The `rabbitmq` receiver reads the monitoring password from a `RABBITMQ_PASSWORD` environment variable, so it never lives in the config file. Every environment below exports to SigNoz Cloud with the same `otlp_http` exporter, so add it once if you do not have one:
+
+```yaml:config.yaml
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-rabbitmq.mdx:74]: Image
Why is there spacing?

File: data/docs/integrations/opentelemetry-rabbitmq.mdx [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+
+Validate, restart, and watch the logs:
+
+```bash
+sudo /usr/bin/otelcol-contrib validate --config /etc/otelcol-contrib/config.yaml
+sudo systemctl restart otelcol-contrib
+sudo journalctl -u otelcol-contrib -f
+```
+
+<Admonition type="info" title="Set up the OpenTelemetry Collector" defaultCollapsed="true">
```
Comment (revmag) [inline data/docs/integrations/opentelemetry-rabbitmq.mdx]: It should come at the top of this section?

**Review by revmag — APPROVED** [review]
